content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,40 @@
 ---
 
 This page describes changes you need to be aware of when upgrading from
-Istio 1.1 to 1.2.  Here, we detail cases where we intentionally broke backwards
+Istio 1.2 to 1.3.  Here, we detail cases where we intentionally broke backwards
 compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
-familiar with the use and operation of Istio 1.1.
+familiar with the use and operation of Istio 1.2.
 
 For an overview of new features introduced with Istio 1.2, please refer
-to the [1.2 release notes](/about/notes/1.2/).
+to the [1.3 release notes](/about/notes/1.3/).
 
 ## Installation and Upgrade
 
-{{< tip >}}
-The configuration model for Mixer has been simplified. Support for
-adapter-specific and template-specific Custom Resources has been
-removed by default in 1.2 and will be removed entirely in 1.3.
-Please move to the new configuration model.
-{{< /tip >}}
-
-Most Mixer CRDs were removed from the system to simplify the configuration
-model, improve performance of Mixer when used with Kubernetes, and improve
-reliability in a variety of Kubernetes environments.
-
-The following CRDs remain:
-
-| Custom Resource Definition name | Purpose |
-| --- | --- |
-| `adapter`| Specification of Istio extension declarations |
-| `attributemanifest` | Specification of Istio extension declarations |
-| `template` | Specification of Istio extension declarations |
-| `handler` | Specification of extension invocations |
-| `rule` | Specification of extension invocations |
-| `instance` | Specification of extension invocations |
-
-In the event you are using the removed mixer configuration schemas, set
-the following Helm flags during upgrade of the main Helm chart:
-`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
+We simplified the configuration model for Mixer and removed support for
+adapter-specific and template-specific Custom Resource Definitions (CRDs)
+entirely in 1.3. Please move to the new configuration model.
+
+We removed the Mixer CRDs from the system to simplify the configuration
+model, improve Mixer's performance in Kubernetes deployments, and improve
+reliability in various Kubernetes environments.
+
+## Secret Service Discovery
+
+In Istio 1.3, we are taking advantage of improvements in Kubernetes to issue
+certificates for workload instances more securely.
+
+Kubernetes 1.12 introduces `trustworthy` JWTs to solve these issues. However,
+support for the `aud` field to have a different value than the API server
+audience didn't become available until [Kubernetes 1.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md).
+To better secure the mesh, Istio 1.3 only supports `trustworthy` JWTs and
+requires the value of the `aud` field to be `istio-ca` when you enable SDS.
+
+Before upgrading your Istio deployment to 1.3 with SDS enabled, verify that you
+use Kubernetes 1.13 or later.
```

Comment:
I wonder should we link the blog post here instead, since it has more information about this change? For example, simply upgrading the k8s cluster to 1.3 doesn't work (it works for GKE only). For OSS k8s, the users will need to do some extra steps. For manged k8s clusters other than GKE, the users need to check with their vendors.